v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -284,7 +284,8 @@ func newGRPCReporter() reporter {
 
 	r.start()
 
-	log.Warningf("AppOptics reporter is initialized: %v", r.done)
+	log.Warningf("AppOptics reporter is initialized. id: %v version: %s.",
```

Comment:
For me, it seems like info (maybe debug is OK).

Reply:
Yes, agree with you that it's a bit too high. However, `WARNING` is the default log level for the time being and here is the only place where the agent could indicate it's up and running.
I would consider proposing `INFO` as the new default level in another Jira ticket and keep these two lines until the default level has been changed.